src/main/java/com/iemr/hwc/controller/CHOApp/CHOAppSyncController.java

```diff
@@ -40,4 +40,14 @@ public ResponseEntity<String> beneficiarySyncToAppLocal(@RequestBody SyncSearchR
         return choappSyncService.getBeneficiaryByVillageIDAndLastModifiedDate(villageIDAndLastSyncDate, Authorization);
     }
 
+    // beneficiary flow status records sync from AMRIT server to CHO app
+    @ApiOperation(value = "Sync beneficiaries flow status records ", consumes = "application/json", produces = "application/json")
+    @RequestMapping(value = { "/benFlowStatusRecordsToLocal" }, method = {
```

Comment:
donot use local in benFlowStatusRecordsToLocal. No one understands local means, they think local to this java service. benFlowStatusRecordsToApp or something you can use

Reply:
oh, got it. Changed.